packages/fxa-settings/src/components/UnitRowSecondaryEmail/index.tsx

```diff
@@ -0,0 +1,81 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+* License, v. 2.0. If a copy of the MPL was not distributed with this
+* file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+import React, { useCallback } from 'react';
+import { useBooleanState } from 'fxa-react/lib/hooks';
+import UnitRow from '../UnitRow';
+import Modal from '../Modal';
+import AlertBar from '../AlertBar';
+
+type UnitRowSecondaryEmailProps = {
+  primaryEmail: string;
+};
+
+export const UnitRowSecondaryEmail = ({ primaryEmail }: UnitRowSecondaryEmailProps) => {
```

Comment:
This component doesn't have any tests yet because it doesn't do anything we aren't testing in other files. When it actually does stuff we'll add tests.